legend-depot-artifacts-refresh/src/test/java/org/finos/legend/depot/store/artifacts/services/TestArtifactsRefreshService.java

```diff
@@ -424,36 +418,19 @@ public void canRefreshProjectMasterVersionWithAllDependenciesTransitively()
 
     }
 
-    @Test
```

Comment:
why is this test deleted

Reply:
because its for versioned, but maybe let me morph it into something it signifies